Objects/regions.c

```diff
@@ -959,3 +961,57 @@ PyTypeObject PyRegion_Type = {
     0,                                       /* tp_alloc */
     PyType_GenericNew,                       /* tp_new */
 };
+
+static const char *get_region_name(PyObject* obj) {
+    if (IS_DEFAULT_REGION(obj->ob_region)) {
+        return "Default";
+    } else if (IS_IMMUTABLE_REGION(obj->ob_region)) {
+        return "Immutable";
+    } else {
+        const regionmetadata *md = (regionmetadata *)obj->ob_region;
+        return md->bridge->name
+            ? PyUnicode_AsUTF8(md->bridge->name)
+            : "<no name>";
+    }
+}
+
+// TODO replace with write barrier code
+int _Pyrona_AddReference(PyObject *tgt, PyObject *new_ref) {
+    if (IS_IMMUTABLE_REGION(new_ref->ob_region)) {
+        // Nothing to do
+        return 1;
```

Comment:
cpython has predefined `true` and `false` values

```suggestion
        return true;
```